src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const priceD = 40;
+  const bigSale = 50;
+  const smallSale = 20;
+  let cost = 0;
+
+  (days >= 7) ? cost = (days * priceD) - bigSale
+    : (days >= 3) ? cost = (days * priceD) - smallSale
+      : cost = days * priceD;
+
```

Comment:
Don't use nested ternary operators, it's bad practice. Rewrite with `if / else` construction.